src/Model_Hydro/CPU_Hydro/CPU_dtSolver_HydroCFL.cpp

```diff
@@ -147,6 +180,7 @@ void CPU_dtSolver_HydroCFL  ( real g_dt_Array[], const real g_Flu_Array[][FLU_NI
 //       no longer used
          MaxCFL = FMAX( CFLx+CFLy+CFLz, MaxCFL );
          */
+#        endif
```

Comment:
How about moving line `161` which is `MaxCFL = FMAX( Us_Max * LorentzFactor_Max + LorentzFactor_s_Max * U_Max, MaxCFL );` to here for clarity?